random/random_test.go

```diff
@@ -75,20 +89,37 @@ func TestParseDistribution(t *testing.T) {
 		},
 	}
 
+	re := regexp.MustCompile("(~?[A-Za-z]+)\\((.*)\\)")
+	repl := "$1:$2"
+
 	for name, cas := range cases {
 		t.Run(name, func(t *testing.T) {
-			got, err := random.ParseDistribution(cas.input)
-			if err != nil {
+			var inputs []string
+			if  altInput := re.ReplaceAllString(cas.input, repl); altInput != cas.input {
```

Comment:
Instead of mutating test cases could we just add more test cases with alternate syntax so it's obvious and readable what we're testing?

Reply:
My reasoning for autogenerating an alternative syntax version of each test is:
* We avoid the chore of copy-paste-edit each test-case.
* We avoid the bloat of the increased number of equivalent tests.
* Tests for the two syntaxes won't get out of sync.

I admit that reading the source it is not obvious that we are testing both syntaxes here, but at least when the test fails, the exact string that failed it will be printed, so whoever debugs a failed test will not try to chase a ghost.

Reply:
Gotcha. What about we *extend* existing test-cases instead?

```diff
        cases := map[string]struct {
                input string
+               alt   string
                want  Distribution
                ok    bool
        }{
                "fixed": {
                        `fixed(1)`,
+                       `fixed:1`,
                        &Fixed{Value: 1},
                        true,
                },
                "uniform": {
                        `uniform(1..10)`,
+                       `uniform:1..10`,
```

Reply:
That could work too.

Reply:
Why a map? Usually we put a slice of structs that gives the fixed order every time. What we try to follow in other projects is this pattern https://github.com/jmoiron/sqlx/blob/master/named_test.go#L9 (note the naming too).

Reply:
The map thing pre-dates this PR so I think it should be handled in a follow-up.

Reply:
I can address this in this PR too, but I'll let you fight this out with the original author @rjeczalik  before I change anything.